javascript/javascript1/week1/smart-ease.js

```diff
@@ -0,0 +1,54 @@
+// Age-ify (A future age calculator)
+
+// year of my birth
+const yearOfBirth = 2000;
+
+// future year
+const  yearFuture = 2058;
+
+// how old will i be
+const age = yearFuture - yearOfBirth;
+console.log("I will be "+ age + " years old in " + yearFuture + "." )
+
```

Comment:
Looks great. Good use of const. 
Programming is always about the details - in the homework description it says to log out a string like the following: 
"You will be 40 years old in 2027", meaning you should change "I" to "you" in line 11 of your code 💫